tests/test_validators_cerberus.py

```diff
@@ -0,0 +1,299 @@
+from __future__ import absolute_import
+import pytest
+
+from spidermon.contrib.validation import CerberusValidator
+from spidermon.contrib.validation import messages
+
+# Format for parameters
+# 1. schema
+# 2. data
+# 3. valid
+# 4. expected_errors
+# 5. Id of test
+
+# Format of test
+# @pytest.mark.parametrize(
+#     "schema,data,valid,expected_errors",
+#     [
+        # pytest.param(
+        #     schema,
+        #     data,
+        #     valid,
+        #     expected,
+        #     id=
+        # )
+#     ],
+# )
+# def test_required_rule(schema, data, valid, expected_errors):
+#     validator = CerberusValidator(schema)
+#     assert validator.validate(data) == (valid, expected_errors)
+
+
+@pytest.mark.parametrize(
+    "schema,data,valid,expected_errors",
+    [
+        pytest.param(
+            {"foo": {"required": True, "type": "string"}, "bar": {"type": "integer"}},
+            {"bar": 1},
+            False,
+            {"foo": [messages.MISSING_REQUIRED_FIELD]},
+            id="REQUIRED - Invalid case",
+        ),
+        pytest.param(
+            {"foo": {"required": True, "type": "string"}, "bar": {"type": "integer"}},
+            {"foo": "toy"},
+            True,
+            {},
+            id="REQUIRED - Valid case"
+        ),
+        pytest.param(
+            {"foo": {}},
+            {},
+            True,
+            {},
+            id="NOT REQUIRED - by default"
+        )
+    ]
+)
+def test_required_rule(schema, data, valid, expected_errors):
```

Comment:
I'm not sure if messages should be validated through the validator.
Probably you should test the translator directly here.
Then, these messages should be asserted in the validator tests.

@rennerocha , any thoughts?